src/items/itemDialog.component.tsx

```diff
@@ -649,6 +649,49 @@ function ItemDialog(props: ItemDialogProps) {
                                 </FormHelperText>
```

Comment:
The tooltips go oval shaped when there are errors
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/29d655b3-dc59-4b63-ab25-ebbff117b9c4)